mllib/src/main/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctions.scala

```diff
@@ -49,6 +49,18 @@ class MLPairRDDFunctions[K: ClassTag, V: ClassTag](self: RDD[(K, V)]) extends Se
       }
     ).mapValues(_.toArray.sorted(ord.reverse))  // This is a min-heap, so we reverse the order.
   }
+
+  def topByKey(num: Int, bucketsCount: Int = 200)(implicit ord: Ordering[V]): RDD[(K, Array[V])] = {
```

Comment:
Doesn't this make a call to `topByKey(n)` ambiguous? maybe not, but seems like it should. I don't think you want a default value here. The 'default' case is the existing method.